README.md

```diff
@@ -78,10 +78,7 @@ jsonschema.Reflect(&TestUser{})
         },
         "tags": {
           "type": "object",
-          "patternProperties": {
-            ".*": {
-              "additionalProperties": true
-            }
```

Comment:
This made no sense. This is the same as simply `"additionalProperties": true`. In fact:

```
{
  "additionalProperties": true
}
```

makes little sense on its own, too. Because it makes sense only in combination with `"type": "object"`.

Reply:
I think the `map[string]interface{}` should be converted to a simple object schema, so `{"type": "object"}` as `additionalProperties` is always `true` by default. The `patternProperties` in the current implementation is a side effect of being able to support element types on a map, e.g. `map[string]string`.